cmd/crd-linter/README.md

```diff
@@ -0,0 +1,51 @@
+# Linter for Kubernetes CustomResourceDefinition resources
+
+The crd-linter encompasses a number of linters that can be run against
+Kubernetes CustomResourceDefinition (CRD) objects to help ensure best-practices
+are followed when CRDs are authored.
+
+## Usage
+
+```
+$ crd-linter --help
+Usage of crd-linter:
+      --exceptions-file string   Path to a list of exceptions for linter failures
+      --linters strings          Full list of linters to run against discovered CustomResourceDefinitions (default [SchemaProvided,PreserveUnknownFields,MaxLengthStrings,MaxItemsArrays])
+      --output-exceptions        If true, an exception list file will be written to the file denoted by '--exceptions-file'
+      --path string              Path to recursively search for CustomResourceDefinition objects
+```
+
+The crd-linter must be pointed at a directory containing YAML files with CRD
+object(s) defined in them using the `--path` flag.
+
+For example:
+
+```
+crd-linter --path ./path/to/crds/
```

Comment:
This project is responsible for gen resources? 
could you add in the first comment why do you think that this project ought to also linter the resources? I think we need here to know when and why it would be required and used? 

Also, would that be a new bin or it would also be hipped with controller-gen? Have we an enhancement proposal for it? and then, why add support to lint only the CRD and not all manifests generated by the project?